tabcmd/commands/extracts/extracts.py

```diff
@@ -0,0 +1,30 @@
+from tabcmd.commands.constants import Errors
+from tabcmd.commands.datasources_and_workbooks.datasources_and_workbooks_command import DatasourcesAndWorkbooks
+from tabcmd.commands.server import Server
+from tabcmd.execution.localize import _
+
+import tableauserverclient as TSC
+
+
+class Extracts(Server):
+    @staticmethod
+    def get_wb_or_ds_for_extracts(args, logger, server):
+        container = Server.get_project_by_name_and_parent_path(
+            logger, server, args.project_name, args.parent_project_path
+        )
+        if args.datasource:
+            logger.debug(_("export.status").format(args.datasource))
+            datasource = Server.get_data_source_item(logger, server, args.datasource, container)
+            return datasource
+
+        elif args.workbook or args.url:
+            logger.info(_("export.status").format(args.workbook))
+            if args.workbook:
+                workbook_item = Server.get_workbook_item(logger, server, args.workbook, container)
```

Comment:
Any reason to add or not to add the `TSC.WorkbookItem` type like we do below in the `else` block?

Reply:
None. I  just don't usually think of it for the return of 'get_workbook_item' and methods with obvious return types like that.